CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.4.4 (next)
 
+* [#39](https://github.com/dblock/iex-ruby-client/pull/39): Fix: remove useless range in dividends api - [@ildarkayumov](https://github.com/ildarkayumov).
```

Comment:
This actually fixes the default range, doesn't it?

Reply:
It adds default range and removes duplicated range use.

Reply:
lets say that